dev/Installer/src/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Windows.CppWinRT" version="2.0.201217.4" targetFramework="native" />
```

Comment:
Hey @EHO-makai  or @BenJKuhn - is there a way to have this solution share wil/cppwinrt packages with the other Project Reunion solutions?

Reply:
I think this involves having a nuget.config at the top level of the repo that solutions share.  I haven't actually done this before so someone will have to prototype the change.  As always, we should log an issue and triage.  I think it is beneficial to make the version of cppwinrt common, at the risk of causing a compat problem between solutions in the project (one solution prefers a different cppwinrt version due to a subtle difference).  But I believe a solution with such an issue can override this with its own nuget.config at its own solution "level".  @BenJKuhn should probably own this issue / scheduling the work for this change.

Reply:
Filed #448